optuna_dashboard/ts/action.ts

```diff
@@ -265,6 +285,44 @@ export const actionCreator = () => {
       })
   }
 
+  const tellTrial = (
```

Comment:
[nit] It's a matter of preference, so please feel free to ignore this suggestion.

I prefer to split this action to avoid `values?: number[]`.

* `make_trial_complete(studyId: number, trialId: number, values: number[])`
* `make_trial_fail(studyId: number, trialId: number)`